### PR TITLE
Event scaffold creation

### LINKED ISSUE #7: Event model creation and migration to DB
We need model, which would represent actual border gaming events. Table events should have following attributes
name :string
description :text
city :string
address :string
start_t :datetime
end_t :datetime
integer :user_id(foreign key)
Validations: We should validate presence and content type of above attributes.

### PR DESCRIPTION
**Pull request by maciek1995**
Removing unnecessary lines from events controller and deleting not needed files, which were created by scaffold

closes #7

### CODE REVIEW

**Comment by hav3loc** [comment]
We'll need to change all views to slim, but it can be automated using, for example, gem erb2slim.

**Comment by hav3loc** [comment]
It all looks good and works fine on my machine, just need to change from erb to slim.

File: spec/controllers/events_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,269 @@
+require 'rails_helper'
+
+RSpec.describe EventsController, type: :controller do
+  let(:user) { User.create!(name: "Username", surname: "Usersurname", city: "Lodz", email: "a@a.a", password: "password")}
```
Comment (bpietraga) [inline spec/controllers/events_controller_spec.rb]: missing space before }

File: app/views/events/index.html.slim [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        td= event.name
+        td= event.description
+        td= event.city
+        td= event.address
+        td= event.start_t
+        td= event.end_t
+        td= link_to 'Show', event
+        td= link_to 'Edit', edit_event_path(event)
+        td= link_to 'Destroy', event, method: :delete, data: { confirm: 'Are you sure?' }
+br/
```
Comment (bpietraga) [inline app/views/events/index.html.slim]: is `/` needed?

File: app/views/events/edit.html.slim [hunk 3]
```diff
@@ -0,0 +1,5 @@
+h1 Editing Event
+= render 'form', event: @event
+= link_to 'Show', @event
+| \
```
Comment (bpietraga) [inline app/views/events/edit.html.slim]: `/` ?

**Comment by bpietraga** [comment]
+1

**Comment by bpietraga** [comment]
+1

**Comment by bpietraga** [comment]
@gotar The event mvc stuff looks ok, but there are unnecessary commits from other PRs here. Maciek says it appeared to after resolving conflicts

**Comment by gotar** [comment]
you need to rebase with origin/master

**Comment by maciek1995** [comment]
@gotar Done rebasing

**Comment by gotar** [comment]
still 16 commits, squash to single one

**Comment by maciek1995** [comment]
@gotar done squashing

File: app/views/events/index.html.slim (hunk 2 above)
Comment (trueskawka) [inline app/views/events/index.html.slim]: not necessary, thx

**Comment by maciek1995** [comment]
@gotar squashed

**Comment by bpietraga** [comment]
+1

**Comment by hav3loc** [comment]
Ready to be checked +1

**Comment by gotar** [comment]
Good enough. +1

**Pull request merged**